rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -800,6 +801,37 @@ int LuaSyncedCtrl::SetAlly(lua_State* L)
 }
 
 
+/*** Changes the start box position of an allyTeam.
+ *
+ * @function Spring.SetAllyTeamStartBox
+ * @number allyTeamID
+ * @number xMin left start box boundary (elmos)
+ * @number zMin top start box boundary (elmos)
+ * @number xMax right start box boundary (elmos)
+ * @number zMax bottom start box boundary (elmos)
+ * @treturn nil
+ */
+int LuaSyncedCtrl::SetAllyTeamStartBox(lua_State* L)
+{
+	const unsigned int allyTeamID = luaL_checkint(L, 1);
+	const float xMin = luaL_checkfloat(L, 2);
+	const float zMin = luaL_checkfloat(L, 3);
+	const float xMax = luaL_checkfloat(L, 4);
+	const float zMax = luaL_checkfloat(L, 5);
+
+	if (!teamHandler.IsValidAllyTeam(allyTeamID))
+		return 0;
+
```

Comment:
```suggestion
	if (xMin < 0.f || zMin < 0.f or xMax > 1.f || zMax > 1.f)
		return 0;
```

Reply:
Sorry I don't follow:
- These are elmos, do you want to check if the elmos are valid?
- If we do the clamping in `TeamHandler.h`, then there should be no need for checking the values after converting them from elmos.

Reply:
ah right, my bad. check in the proper unit of course, or let it through if it clamps.